sync-core/src/main/java/com/cloudant/sync/datastore/BasicDatastore.java

```diff
@@ -921,25 +943,20 @@ private long insertRevision(SQLDatabase db,InsertRevisionOptions options) {
             logger.fine("New revision inserted: " + options.docNumericId + ", " + options.revId);
             newSequence = db.insert("revs", args);
             if (newSequence < 0) {
-                throw new IllegalStateException("Unknown error inserting new updated doc, please checking log");
+                throw new IllegalStateException("Unknown error inserting new updated doc, please check log");
```

Comment:
Hopefully the SQL library will have logged an error. Right now we probably can't say something much better.